docs/modules/reference/pages/service-assurance/monitors/MinaSshMonitor.adoc

```diff
@@ -0,0 +1,75 @@
+
+= MinaSshMonitor
+
+The MinaSshMonitor tests the availability of an SSH service.
+During the poll, it attempts to connect on the specified port and then authenticate in.
+If the authentication is successful, then the service is considered up.
+Optionally, a remote command to execute can be provided and if that is successful, the service is considered up.
+
```

Comment:
Add a short statement to make it easy to see the difference between this and the standard Ssh Monitor.
```suggestion

== MinaSshMonitor vs SshMonitor

The MinaSshMonitor tests connecting and authenticating to a node via SSH.
The SshMonitor only tests that the SSH port is accepting connections.

```

On the SshMonitor.adoc page, add a similar statement in reverse order:
```
== SshMonitor vs MinaSshMonitor 

The SshMonitor only tests that the SSH port is accepting connections.
The MinaSshMonitor can be used to test both connecting and authenticating to a node via SSH.